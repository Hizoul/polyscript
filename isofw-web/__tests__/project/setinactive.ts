import setinactive from "isofw-shared/src/tests/project/setinactive"
import ProjectActiveButton from "isofw-web/src/components/project/activityButton"

setinactive(ProjectActiveButton)
