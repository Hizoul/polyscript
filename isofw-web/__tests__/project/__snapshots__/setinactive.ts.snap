// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`free unused presets after program Update works as planned: Before anything 1`] = `
<div
  className="Button"
>
  {
  "text": "Re-activate",
  "iconFa": "step-backward"
}
</div>
`;

exports[`free unused presets after program Update works as planned: after All having been freed up again 1`] = `
<div
  className="Button"
>
  {
  "text": "Re-activate",
  "iconFa": "step-backward",
  "loading": false
}
</div>
`;

exports[`free unused presets after program Update works as planned: after reservation of four presets 1`] = `
<div
  className="Button"
>
  {
  "text": "Deactivate",
  "iconFa": "step-backward",
  "loading": false
}
</div>
`;
